charts/ocis/values.yaml

```
@@ -935,6 +935,11 @@ services:
   postprocessing:
     # -- Per-service resources configuration. Overrides the default setting from `resources` if set.
     resources: {}
+    # -- Per-service store configuration for the eventhistory service. Overrides the default setting from `store` if set.
+    store:
+      {}
```

Minor thing: maybe the formatting should be similar to the others
```suggestion
    store: {}
```